cpp/include/cudf/stream_compaction.hpp

```diff
@@ -283,11 +278,11 @@ std::unique_ptr<table> unordered_drop_duplicates(
  * @param[in] null_handling flag to include or ignore `null` while counting
  * @param[in] nan_handling flag to consider `NaN==null` or not
  *
- * @return number of distinct consecutive groups in the column
+ * @return number of consecutive groups of equivalent rows in the column
  */
-cudf::size_type distinct_count(column_view const& input,
-                               null_policy null_handling,
-                               nan_policy nan_handling);
+cudf::size_type unique_count(column_view const& input,
+                             null_policy null_handling,
+                             nan_policy nan_handling);
 
 /**
  * @brief Count the number of consecutive groups of equivalent elements in a table.
```

Comment:
Docstring parameters need fixed to avoid using "distinct" in the `unique_count` definition:
```diff
- * @param[in] input Table whose number of distinct consecutive groups will be counted
+ * @param[in] input Table whose consecutive groups of equivalent rows will be counted
```